src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PreviewWindow.xaml

```diff
@@ -64,7 +64,8 @@
       BorderThickness="1">
       <ScrollViewer
         HorizontalScrollBarVisibility="Auto"
-        IsTabStop="True">
+        IsTabStop="True"
```

Comment:
Was this still needed?

Reply:
Yes. That makes the narrator to read a more meaningful name for the Preview Changes Pane